### PR TITLE
Added fixes and installation documentation

### LINKED ISSUE #69: Insufficient Doc for Desktop version of Pslab
**Describe the issue**

Currently there are two instruction in using pslab desktop.. first is connecting and then opening.. 🤷‍♂ which isn't really helpful

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Sid911**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
- [ ] Bug fix
- [ ] Feature implementation
- [x] Doc updates

* **What changes have you introduced?**
- fixes #69 
- Added installation documentation for all three platforms
- fixed the erros in documentation when this file was changed from rst to md

* **Preview / Steps to verify your work**:
image

- Go to https://github.com/sid911/pslab-documentation 
- Click on Desktop.md file to preview it

### CODE REVIEW

**Comment by MuhammadWasif** [comment]
This is for developers. Isn't there a way to install production build? 🤔🤔

**Comment by Sid911** [comment]
nope... @MuhammadWasif

**Comment by Sid911** [comment]
yeah there is... its in install branch but that never gets builds because build system is broken

**Comment by Sid911** [comment]
@fcartegnie can this be reviewed? thanks

File: desktop.md [hunk 1]
```diff
@@ -1,12 +1,16 @@
 # Desktop Application
 
-## How to install the Desktop Application
+Dowload the Stable version for various platforms here https://github.com/fossasia/pslab-desktop/tree/install 🌐
```
Comment (fcartegnie) [inline desktop.md]: No second commit to fix your first one

Comment (Sid911) [reply desktop.md]: sure I will squash them.. 😇

**Comment by Sid911** [comment]
@fcartegnie Done :)

File: desktop.md [hunk 2]
```diff
@@ -1,15 +1,39 @@
-Desktop Application
-===================
+# Desktop Application
 
-How to use the Desktop App
---------------------------
+Dowload the Stable version for various platforms here https://github.com/fossasia/pslab-desktop/tree/install 🌐
```
Comment (fcartegnie) [inline desktop.md]: typo

File: desktop.md [hunk 3]
```diff
@@ -1,15 +1,39 @@
...
--   To use any feature of the PSLab Desktop application first follow
+## How to install the Desktop Application (latest Development version)
+
+- Windows  _(currently there is no official distribution for latest dev pslab Desktop Application in order to run the application you need to do the following things)_:
+    
+    1. Go to offical Repository at https://github.com/fossasia/pslab-desktop 
+    2. Dowload as a zip or Clone in next step
+        1. Click on Clone the download it as zip
+        2.  Extract the zip file
+    3. Open terminal and type ```git clone https://github.com/fossasia/pslab-desktop.git```
```
Comment (fcartegnie) [inline desktop.md]: might want formatting like npm commands

Comment (Sid911) [reply desktop.md]: Done :) if you meant formatting like this 
image

File: desktop.md (hunk 2 above)
Comment (Sid911) [reply desktop.md]: fixed :)

**Comment by Sid911** [comment]
@fcartegnie fixed type and added formatting, if that meant this 
image 😅

**Comment by Sid911** [comment]
@cweitat please review and merge 😀

File: desktop.md [hunk 4]
```diff
@@ -1,15 +1,42 @@
...
+    
+        ```
+        npm install
+        ```
+        _Wait for it to finish_
+        ```
+        npm start
+        ```
+
+    7. PSLab Desktop Should Open in front of you 😇
```
Comment (cweitat) [inline desktop.md]: Keep the capitalization of the words constant

File: desktop.md [hunk 5]
```diff
@@ -1,15 +1,42 @@
...
-How to use the Desktop App
---------------------------
+Download the Stable version for various platforms here https://github.com/fossasia/pslab-desktop/tree/install 🌐
 
--   To use any feature of the PSLab Desktop application first follow
+## How to install the Desktop Application (latest Development version)
+
+- Windows  _(currently there is no official distribution for latest dev pslab Desktop Application in order to run the application you need to do the following things)_:
+    
+    1. Go to offical Repository at https://github.com/fossasia/pslab-desktop
```
Comment (cweitat) [inline desktop.md]: Keep the capitalization of the words constant

File: desktop.md (hunk 4 above)
Comment (Sid911) [reply desktop.md]: Sure ... Working on it 😇

File: desktop.md [hunk 6]
```diff
@@ -1,15 +1,42 @@
...
---------------------------
+Download the Stable version for various platforms here https://github.com/fossasia/pslab-desktop/tree/install 🌐
 
--   To use any feature of the PSLab Desktop application first follow
+## How to install the Desktop Application (latest Development version)
+
+- Windows  _(currently there is no official distribution for latest dev pslab desktop application in order to run the application you need to do the following things)_:
+    
+    1. Go to offical Repository at https://github.com/fossasia/pslab-desktop 
+    2. Dowload as a zip or clone in next step
```
Comment (cweitat) [inline desktop.md]: download . typo

File: desktop.md [hunk 7]
```diff
@@ -1,15 +1,42 @@
...
-How to use the Desktop App
---------------------------
+Download the Stable version for various platforms here https://github.com/fossasia/pslab-desktop/tree/install 🌐
 
--   To use any feature of the PSLab Desktop application first follow
+## How to install the Desktop Application (latest Development version)
+
+- Windows  _(currently there is no official distribution for latest dev pslab desktop application in order to run the application you need to do the following things)_:
+    
+    1. Go to offical Repository at https://github.com/fossasia/pslab-desktop
```
Comment (cweitat) [inline desktop.md]: typo official

**Comment by Sid911** [comment]
@cweitat I think that's it.. for typos that I could spot 😅

File: desktop.md [hunk 8]
```diff
@@ -1,15 +1,42 @@
...
+
+- To use any feature of the PSLab Desktop application first follow
     these 2 steps:
     1.  Connect PSLab board with your Desktop using USB cable
-    2.  Open \<a
-        href="<https://github.com/fossasia/pslab-desktop>"\>PSLab
-        Desktop application\</a\>
+    2.  Open [PSLab Desktop application](https://github.com/fossasia/pslab-desktop)
 
 You will be presented with a home screen like this.
```
Comment (cweitat) [inline desktop.md]: like what? where's the home screen?

Comment (Sid911) [reply desktop.md]: @cweitat  that was there from before.... I just changed the link which was present from before

Comment (Sid911) [reply desktop.md]: image
in below line there is the image

Comment (Sid911) [reply desktop.md]: I didn't even change this section @cweitat

**Comment by Sid911** [comment]
@cweitat that section is already made.. just the link wasn't correct as it was changed from .rst to .md,I changed just that part :) 
image

**Pull request merged**